base.go

```diff
@@ -5,11 +5,11 @@ import "time"
 // Base structure contains fields that are common to objects returned by the Omise's REST
 // API.
 type Base struct {
-	Object   string    `json:"object"`
-	ID       string    `json:"id"`
-	Live     bool      `json:"livemode"`
-	Location *string   `json:"location"`
-	Created  time.Time `json:"created"`
```

Comment:
Thanks @thetkpark but then this would cause a similar issue for users of the library below the current latest version 🤔